packages/@sanity/cli/src/commands/init/initCommand.ts

```diff
@@ -22,6 +26,7 @@ Options
   --project-plan <name> Optionally select a plan for a new project
   --coupon <name> Optionally select a coupon for a new project (cannot be used with --project-plan)
   --no-typescript Do not use TypeScript for template files
+  --package-manager <manager> Specify which package manager to use [allowed: ${allowedPackageManagersString}]
```

Comment:
Tiny: would `<name>` make more sense than `<manager>` here? It makes it consistent with the other flags.

Reply:
```suggestion
  --package-manager <name> Specify which package manager to use [allowed: ${allowedPackageManagersString}]
```